<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tagAdder</class>
 <widget class="QWidget" name="tagAdder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>325</width>
    <height>231</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>108</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>108</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="styleSheet">
         <string notr="true">font-size:14px;
color: white;
font-family: 'Crete Round';</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>61</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="tagTextTitle">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Tag text (max 6 characters):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="tagTextEdit">
           <property name="maximumSize">
            <size>
             <width>112</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background:#2B3663;
	font-family: 'Crete Round';</string>
           </property>
           <property name="maxLength">
            <number>6</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <property name="styleSheet">
         <string notr="true">font-size:14px;
color: white;
font-family: 'Crete Round';</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="tagColorLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Tag Color:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="tagColorComboBox">
           <property name="minimumSize">
            <size>
             <width>106</width>
             <height>31</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>106</width>
             <height>31</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background:white;
color:#699EFF;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="orLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Or</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="tagColorEdit">
           <property name="minimumSize">
            <size>
             <width>110</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>110</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background:#2B3663;
	font-family: 'Crete Round';</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="previewText">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Crete Round</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;

	font-family: 'Crete Round';</string>
        </property>
        <property name="text">
         <string>--Preview--</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="tagPreviewBox" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="tagPVcentreBox" native="true">
           <property name="maximumSize">
            <size>
             <width>91</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_7" native="true">
        <property name="styleSheet">
         <string notr="true">QWidget{
	font-size:14px;
	color: white;
	border-radius:5px;
	
}
QPushButton{
	background:#2B3663;
	font-family: 'Crete Round';
}

QPushButton:hover{
	background:rgb(86, 99, 152);
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="confirmButton">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>22</height>
            </size>
           </property>
           <property name="text">
            <string>Confirm</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancelButton">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>22</height>
            </size>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
